lib/src/controls/better_player_material_controls.dart

```diff
@@ -629,6 +640,62 @@ class _BetterPlayerMaterialControlsState
     );
   }
 
+  Widget _buildRestart(VideoPlayerController controller) {
+    return BetterPlayerMaterialClickableWidget(
+      key: const Key("better_player_material_controls_restart_button"),
+      onTap: () {
+        //todo functionality
```

Comment:
czemu tutaj todo?

Reply:
dałem tod bo z api dostajemy sessions jako live i nie moge zabardzo zrobić restart tego video, merapar ma to poprawić dlatego nie pisałem funkcjonalności tylko przygotowałem buttony a jak to poprawi merapar to to dopisze i przetestuje